src/bindings/dom.js

```diff
@@ -13,6 +13,7 @@ export function setAttributes(element, id, args) {
   if (args) {
     element.setAttribute('data-l10n-args', JSON.stringify(args));
   }
+  return element;
```

Comment:
It allows me to chain that into JS code like this: https://github.com/zbraniecki/gecko-dev/blob/2e14a3a53d6476002f3c4f79e6871376c6056b1e/toolkit/content/aboutSupport.js#L479